minecraft_servers/nitrox/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "server_name": "nitrox",
+    "nice_name": "NitroX",
+    "direct_ip": "play-nitrox.net",
+    "social": {
+      "tiktok": "nitroxnet",
+      "instagram": "nitroxnet",
+      "teamspeak": "ts.nitroxnetwork.eu",
+      "discord": "https://discord.gg/MU8kM7vYpJ"
+    },
+    "discord": {
+      "server_id": 966020442571808848,
+      "rename_to_minecraft_name": true

```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.